examples/client/Locomotion/src/pages/Profile/Phone.js

```diff
@@ -44,7 +44,7 @@ const Phone = ({ navigation }) => {
   };
   useEffect(() => {
     fetchHideCaptchaSetting();
-  }, []);
+  }, [user?.phoneNumber]);
```

Comment:
problem here is that each change to the phone number (e.g from 052 to 0524) this effect will run
can we use other dependency?